config/tailwind.config.js

```diff
@@ -8,6 +8,7 @@ module.exports = {
     "./node_modules/flowbite/**/*.js",
     "./test/components/previews/**/*",
     "./test/components/viral/system/**/*",
+    "./config/locales/en.yml",
```

Comment:
Can you remove this line, I don't think tailwind needs access to the translations

Reply:
We actually do need it in there.  It allows for doing things like on line 408 and 411.  Without having tailwind scan these we cannot have this mark up.  Doing it another way would be extremely complicated.

Reply:
Just thought of something, I will try tomorrow morning

Reply:
Your right, better solution is just to create a custom css class.  See 58bd686